contracts/LockedNORI.sol

```diff
@@ -138,7 +138,7 @@ contract LockedNORI is
     uint256 amount,
     bytes calldata userData,
     bytes calldata operatorData
-  ) external override {
+  ) external override whenNotPaused {
```

Comment:
this is redundant since `_beforeTokenTransfer` uses `whenNotPaused`

Reply:
Hmm, yeah you're right.  I was thinking there was a path through _depositFor that would still mutate state and create a schedule but as long as it also sent tokens the `_beforeTokenTransfer` hook would be called.